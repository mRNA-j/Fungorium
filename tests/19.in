create_mushroom_picker Bob Mp1
create_entomologist John El1
next_round
statRound