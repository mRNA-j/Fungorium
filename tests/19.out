ActivePlayer: Bob
Current round: 2
